db/migrate/20230504172751_create_bookings.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBookings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :bookings do |t|
+      t.references :session, null: false, foreign_key: true
+      t.references :user, null: false, foreign_key: true
+      t.boolean :canceled, null: false, default: false
+
+      t.timestamps
+    end
+    change_column :bookings, :canceled, :boolean, null: false, default: false
+  end
+end
```

Comment:
change_column is used when you update an existing table in a new migration file.
In this case you are create the table and you already set the column to be ` t.boolean :canceled, null: false, default: false` so what does change_column do ?

Reply:
Just to be safe I think it's always best to drop then create again ad migrate when modifying a migration file

Reply:
got it. on it.